.gitignore

```diff
@@ -5,6 +5,5 @@ dist
 node_modules
 svn
 .phpunit.result.cache
-hovercards.css
-hovercards.js
```

Comment:
I think we don't need these files. Please let me know if we need them.

Reply:
Yep, we don't need them anymore as we're now bundling.